v22.2/backup-and-restore-overview.md

```diff
@@ -20,20 +20,19 @@ For an explanation of how a backup works, see [Backup Architecture](backup-archi
-[Encrypted backup and restore](take-and-restore-encrypted-backups.html)      | An encrypted backup using a KMS or passphrase.                                                                                                               | {{ site.data.products.dedicated }}<br>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)
-[Locality-aware backup and restore](take-and-restore-locality-aware-backups.html) | A backup where each node writes files only to the backup destination that matches the node locality configured at node startup.                              | {{ site.data.products.dedicated }}<br>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)
-[Scheduled backup](manage-a-backup-schedule.html)                  | A schedule for periodic backups.                                                                                                                             | {{ site.data.products.dedicated }}<br>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)  
-[Automated backup](../cockroachcloud/backups-page.html)                 | Cockroach Labs runs full backups daily and incremental backups hourly for every CockroachDB Cloud cluster.                                                   | {{ site.data.products.db }} clusters    
+Backup / Restore  | Description  | Product Support | CockroachDB Cloud Backup Type Support
+------------------+--------------+-----------------+------------------------
+[Full backup](take-full-and-incremental-backups.html) | An un-replicated copy of your cluster, database, or table's data. A full backup is the base for any further backups. | All products (Enterprise license not required) | <ul><li>Customer-owned backups</li><li>Managed-service backups</li></ul>
+[Incremental backup](take-full-and-incremental-backups.html) | A copy of the changes in your data since the specified base backup (a full or full and incremental). | <ul><li>{{ site.data.products.serverless }}</li><li>{{ site.data.products.dedicated }}</li><li>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)</li></ul> | <ul><li>Customer-owned backups</li><li>Managed-service backups (dedicated clusters only)</li></ul>
+[Scheduled backup](manage-a-backup-schedule.html) | A schedule for periodic backups. | <ul><li>{{ site.data.products.serverless }}</li><li>{{ site.data.products.dedicated }}</li><li>{{ site.data.products.core }} with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html)</li></ul> | Customer-owned backups
```

Comment:
@livlobo I haven't mentioned here that dedicated managed-service backups run off of scheduled backups and therefore GC TTL and backups are decoupled. Not sure if we want to mention here, perhaps a little much for this table. Could add a note beneath the table to relay this information for v22.2

Reply:
Note sounds good. Let's wait for the change to get rolled out to existing clusters though.

Reply:
Yes, OK, I've opened another issue for that https://cockroachlabs.atlassian.net/browse/DOC-6654 so I can move this PR along.